app/controllers/categories_controller.rb

```diff
@@ -11,6 +11,6 @@ def show
 
     @room = ChatRoom.includes(chat_messages: [:user]).find(@category.chat_room_id) if @category.chat_room_id
 
-    @doping_publications = Doping.category.take.publications.active.where(category_id: @category.id)
+    @doping_publications = DopingPublication.where(doping_id: 2, is_active: true).order(created_at: :desc)
```

Comment:
Bunun view kısmında döngü içinde .publication şeklinde çağrılıyor. Bu tarz durumlar için includes ile eager loading durumunu gerçekleyelim.
```suggestion
    @doping_publications = DopingPublication.includes(:publication).where(doping_id: 2, is_active: true).order(created_at: :desc)
```